app/Traits/ExportableTrait.php

```diff
@@ -31,10 +31,14 @@ public function export(): string
         // Entity values
         if (!empty($this->entity)) {
             // Todo: put these in with()
-            $foreigns = ['posts', 'relationships', 'abilities', 'events', 'tags', 'assets', 'entityAttributes'];
+            $foreigns = ['posts', 'relationships', 'abilities', 'events', 'tags', 'assets', 'entityAttributes', 'image', 'header'];
```

Comment:
Missing eager loading to not crash mysql on a large campaign :-) You can temporarily set `APP_LAZY=true` in your .env to see what needs to be added.